### PR TITLE
Change event polling interval

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

File: src/config/constants.ts [hunk 1]
```diff
@@ -14,7 +14,8 @@ export const OPENSHIFT_VERSION_OPTIONS: OpenshiftVersionOptionType[] = [
 export const CLUSTER_MANAGER_SITE_LINK =
   'https://cloud.redhat.com/openshift/install/metal/user-provisioned';
 
-export const POLLING_INTERVAL = 10000;
+export const POLLING_INTERVAL = 10 * 1000;
```
Comment (mareklibra) [inline src/config/constants.ts:17]: A cluster polling interval should be increased too. What about 60 seconds?

File: src/config/constants.ts [hunk 2]
```diff
@@ -14,7 +14,8 @@ export const OPENSHIFT_VERSION_OPTIONS: OpenshiftVersionOptionType[] = [
...
+export const EVENTS_POLLING_INTERVAL = 60 * 1000;
```
Comment (mareklibra) [inline src/config/constants.ts:18]: I am not sure about such long interval. I think 30 seconds would be reasonable delay. @jtomasek , wdyt?

Comment (jtomasek) [reply src/config/constants.ts:18]: IMO the cluster polling interval should be kept at 10s, we need to keep the UI up to date - state changes etc.
60s for events is probably fine

Comment (mareklibra) [reply src/config/constants.ts:18]: Ok, changing the cluster to 10 secs to keep the rendered info up-to-date.

I still think we should decrease the time for event polling at least once we move the events to a modal.

**Pull request merged**